### PR TITLE
Change Attendee.AdminNotes field collation to allow unicode

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3061395&fileview=grid

Signed-off-by: romanetar <roman_ag@hotmail.com>

### CODE REVIEW

File: database/migrations/model/Version20221012133111.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class Version20221012133111 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $sql = <<<SQL
+ALTER TABLE SummitAttendee MODIFY AdminNotes VARCHAR(1024) CHARACTER SET utf8 COLLATE utf8_unicode_ci NULL;
```
Comment (smarcet) [inline database/migrations/model/Version20221012133111.php:16]: this is not correct
check the collation of the DB
utf8mb4 ( there are Chinese kanjis that needs 4 bytes)
regards

File: tests/OAuth2AttendeesApiTest.php [hunk 2]
```diff
@@ -240,6 +240,43 @@ public function testUpdateAttendee($summit_id = 27, $attendee_id = 18502){
...
+            [],
+            [],
+            $headers,
+            json_encode($data)
+        );
+
+        $content = $response->getContent();
+        $this->assertResponseStatus(201);
+        $attendee = json_decode($content);
+        $this->assertTrue(!is_null($attendee));
```
Comment (smarcet) [inline tests/OAuth2AttendeesApiTest.php:276]: u need to check that admin notes are the same that u provided

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: database/migrations/model/Version20221012133111.php (hunk 1 above)
Comment (smarcet) [reply database/migrations/model/Version20221012133111.php:16]: also please check that migration is compatible with mysql 5.7

Comment (romanetar) [reply database/migrations/model/Version20221012133111.php]: fixed

File: tests/OAuth2AttendeesApiTest.php (hunk 2 above)
Comment (romanetar) [reply tests/OAuth2AttendeesApiTest.php:278]: check added

**Comment by romanetar** [comment]
@smarcet review resolved here https://github.com/OpenStackweb/summit-api/commit/954d95560f66b56edcbdc9bde3f613e7bcabf1c5

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**